src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -2622,6 +2622,68 @@ public void ParseReferences()
             // TODO: multiple files, quotes, etc.
         }
 
+        [Fact]
```

Comment:
Consider linking the work item in the new tests.

```suggestion
        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/71022")]
```